File: automateboringstuff/squaretriangle.py
count = 6
while count < 10000:
	for i in range(0,round((count**2)/2)):
		if count**2 == ((i*(i+1))/2):
			print("{}; n = {}, m = {}".format(count**2,count,i))
		else:
			continue
	count += 1
			

File: automateboringstuff/squaretriangle.java
public class squaretriangle{
	public static void squareNumbers(double limit){
		int found = 0;
		double count = 1;
		double count_squared = (count*count);
		while(true){
			if(found == limit){
				break;
			}
			for(double i = 0; i < Math.round(count_squared/2); i++){
				double tri = ((i*(i+1))/2);
				if(count_squared == tri){
					System.out.println(String.format("%d; m= %d, n= %d",count_squared,count,i));
					found++;
				}else{
					continue;
				}
			}
			count++;
		}
	}
	
	public static void main(String args[]){
		squareNumbers(50);
	}
}